src/core/broadcaster.ts

```diff
@@ -92,7 +92,7 @@ export module Core {
      * @returns {Broadcaster} The calling Broadcaster
      */
     public deregisterAllListeners() {
-      this._key2callback = new Utils.StrictEqualityAssociativeArray();
```

Comment:
I believe broadcasters push in a string as a key right?  Up to you if you want to enforce this since Broadcasters are going away.